src/Raven.Server/Documents/Handlers/ReplicationHandler.cs

```diff
@@ -236,10 +235,64 @@ public Task Performance()
             return true;
         }
 
+        [RavenAction("/databases/*/replication/pulses/live", "GET", AuthorizationStatus.ValidUser, SkipUsagesCount = true)]
+        public async Task PulsesLive()
```

Comment:
maybe stick to one word? 'Heartbeat'? It is a bit confusing that we are naming same thing using two different words.